requirements-dev.txt

```diff
@@ -126,8 +125,6 @@ ruamel-yaml==0.18.6
     # via ansible-lint
 ruamel-yaml-clib==0.2.8
     # via ruamel-yaml
-selinux==0.3.0
```

Comment:
You'll need to re-compile on Linux to avoid this package being removed.

Reply:
See 02777f7eb908c354345996923c5f94e7dc8bc351.